internal/olm/operator/registry/olm_resources.go

```diff
@@ -48,6 +48,18 @@ func withPackageChannel(pkgName, channelName, startingCSV string) func(*v1alpha1
 	}
 }
 
+// withInstallPlanApproval sets the Subscription's install plan approval field
+// to manual
+func withInstallPlanApproval(approval v1alpha1.Approval) func(*v1alpha1.Subscription) {
+	return func(sub *v1alpha1.Subscription) {
+		if sub.Spec == nil {
+			sub.Spec = &v1alpha1.SubscriptionSpec{}
+		}
+		// set the install plan approval to manual
```

Comment:
This comment is different than what this function is actually doing. Probably just remove it.
```suggestion
```